modules/solid_mechanics/src/materials/NewtonIterationSingleVariable.C

```diff
@@ -0,0 +1,443 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "NewtonIterationSingleVariable.h"
+
+#include "InputParameters.h"
+#include <cmath>
+#include "Conversion.h"
+
+template <>
+InputParameters
+validParams<NewtonIterationSingleVariable>()
+{
+  InputParameters params = emptyInputParameters();
+
+  // Newton iteration control parameters
+  params.addParam<unsigned int>("max_its", 200, "Maximum number of Newton iterations");
```

Comment:
While we are breaking BISON, could we change the variable name from _max_its to _max_iterations?